bin/doc

```diff
@@ -100,14 +105,17 @@ cat > target/doc/index.html <<EOF
 </html>
 EOF
 
-# Make the logo link to the nice homepage we just created. Otherwise it just
-# links to the root of whatever crate you happen to be looking at.
-cat >> target/doc/main.js <<EOF
+    # Make the logo link to the nice homepage we just created. Otherwise it just
+    # links to the root of whatever crate you happen to be looking at.
+    cat >> target/doc/main.js <<EOF
 ;
 var el = document.querySelector("img[alt=logo]").closest("a");
 if (el.href != "index.html") {
     el.href = "../index.html";
 }
 EOF
 
-echo "Docs are in target/doc/index.html"
+    echo "Docs are in target/doc/index.html"
+}
+
+main "$@"
```

Comment:
What's going on here? Seems like you disabled a bunch of the script by putting it in an unused function?